packages/webview_flutter/webview_flutter_android/android/src/test/java/io/flutter/plugins/webviewflutter/PluginBindingFlutterAssetManagerTest.java

```diff
@@ -0,0 +1,54 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package io.flutter.plugins.webviewflutter;
+
+import static org.junit.Assert.assertArrayEquals;
+import static org.junit.Assert.fail;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import android.content.res.AssetManager;
+import io.flutter.embedding.engine.plugins.FlutterPlugin.FlutterAssets;
+import io.flutter.plugins.webviewflutter.FlutterAssetManager.PluginBindingFlutterAssetManager;
+import java.io.IOException;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mock;
+
+public class PluginBindingFlutterAssetManagerTest {
+  @Mock AssetManager mockAssetManager;
+  @Mock FlutterAssets mockFlutterAssets;
+
+  PluginBindingFlutterAssetManager tesPluginBindingFlutterAssetManager;
+
+  @Before
+  public void setUp() {
+    mockAssetManager = mock(AssetManager.class);
+    mockFlutterAssets = mock(FlutterAssets.class);
+
+    tesPluginBindingFlutterAssetManager =
+        new PluginBindingFlutterAssetManager(mockAssetManager, mockFlutterAssets);
+  }
+
+  @Test
+  public void list() {
+    try {
+      when(mockAssetManager.list("test/path"))
+          .thenReturn(new String[] {"index.html", "styles.css"});
+      String[] actualFilePaths = tesPluginBindingFlutterAssetManager.list("test/path");
+      verify(mockAssetManager).list("test/path");
+      assertArrayEquals(new String[] {"index.html", "styles.css"}, actualFilePaths);
+    } catch (IOException ex) {
+      fail();
+    }
+  }
```

Comment:
There should be a test ensuring that a null return from `AssetManager.list` is handled correctly. Especially since that will need to be converted to an empty list at the Pigeon API boundary.

Reply:
I have added the test `` to the FlutterAssetManagerHostApiImplTest.java in my latest commit (
08023b2). The `PluginBindingFlutterAssetManager` and `RegistrarFlutterAssetManager` implementations simply relays the result of the `AssetManager.list` method to the `FlutterAssetManagerHostApiImpl` class. The later is responsible for preparing the result in a way that it can be send on the method channel.

This was a good catch though, it found a bug that I overlooked. Many thanks.